libcontainer/configs/config.go

```diff
@@ -33,9 +33,10 @@ type Seccomp struct {
 type Action int
 
 const (
-	Kill Action = iota - 4
```

Comment:
just like the define of "EqualTo Operator", I do think it has problem,Maybe the error caused by other. It doesn't matter all the same,just a surgestion. thanks for your reply.